.github/workflows/ci-backend.yml

```diff
@@ -126,23 +126,15 @@ jobs:
                 group: [1, 2, 3, 4, 5]
                 include:
                     # :TRICKY: Run FOSS tests in a separate container
-                    - python-version: '3.8.5'
```

Comment:
We had an additional test against 3.9 but I don't think it should be required.

Reply:
Yeah it's not like we're a library that people want to be importing I don't think?

The main use case is using our built docker image. Maybe there's value in validating if we're incompatible with newer versions though.

Fine to remove though, it seems like a distraction.